### PR TITLE
feat: different display for owner and participate

### PR DESCRIPTION
**Pull request by pastezzz**

### CODE REVIEW

File: src/pages/dashboard.tsx [hunk 1]
```diff
@@ -30,13 +30,16 @@ interface Event {
...
+  ownerID?: string;
   // Add other properties as needed
 }
 interface EventCardProps {
   event: Event;
+  myEvent: boolean
 }
 
-const EventCard: React.FC<EventCardProps> = ({ event }) => {
+const EventCardMyEvent: React.FC<EventCardProps> = ({ event, myEvent }) => {
```
Comment (louisunlimited) [inline src/pages/dashboard.tsx:42]: Can you change this back to `EventCard`?

File: src/pages/dashboard.tsx [hunk 2]
```diff
@@ -265,18 +269,6 @@ const NewEventCard = () => {
...
-
-  if (status === "loading") {
-    return <div>Loading...</div>;
-  }
-
-  if (!session) {
-    void router.push("/");
-    return <div>Redirecting...</div>;
-  }
-
```
Comment (louisunlimited) [inline src/pages/dashboard.tsx:279]: These are new features, don't delete these

File: src/pages/event/[id].tsx [hunk 3]
```diff
@@ -36,11 +36,11 @@ interface EditDialogProps {
...
-  isOpen,
-  close,
-  eventName,
-  onSubmit,
-}) => {
+                                                 isOpen,
+                                                 close,
+                                                 eventName,
+                                                 onSubmit,
+                                               }) => {
```
Comment (louisunlimited) [inline src/pages/event/[id].tsx:43]: Can you run prettier lint on this ?

File: src/pages/event/[id].tsx [hunk 4]
```diff
@@ -133,11 +133,11 @@ interface DeleteDialogProps {
...
-  close,
-  eventName,
-  onSubmit,
-}) => {
+                                                     isOpen,
+                                                     close,
+                                                     eventName,
+                                                     onSubmit,
+                                                   }) => {
   return (
```
Comment (louisunlimited) [inline src/pages/event/[id].tsx:141]: Or just generally run a lint

**Review by louisunlimited — CHANGES_REQUESTED** [review]

**Review by RaymondWHZ — APPROVED** [review]
Nice feature!

**Pull request merged**